source/index.html

```diff
@@ -1,12 +1,182 @@
+      <p class="example__intro">Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на Cat Energy Slim.
+        Отличный результат без изнуряющих
+        тренировок! При этом он не менял своих привычек и по-прежнему спит по 16 часов в день.</p>
+      <dl class="example__param">
+        <dt class="example__param-weight">5 кг</dt>
+        <dd class="example__param-description"> <small>Снижение веса</small></dd>
+      </dl>
+      <dl class="example__param">
+        <dt class="example__param-weight">60 дней</dt>
+        <dd class="example__param-description"> <small>Затрачено времени</small></dd>
+      </dl>
+      <p class="example__price">Затраты на питание: 15 000 РУБ.</p>
+      <img class="example__cat-img" src="img/fat-cat.png" width="280" height="232"
+        alt="Изменения кота после смены питания.">
+      <!-- не уверена, что эта картинка контентная, т.к. она видоизменяется непонятным образом -->
+      <div class="example__range-scale">
+        <span class="example__range-before">Было</span>
+        <div class="example__range-bar">
+          <button class="example__range-toggle">
+            <span class="visually-hidden">Изменить кота на картинке.</span>
+          </button>
+          <span class="example__range-after">Стало</span>
+        </div>
+      </div>
+    </section>
+  </main>
+  <aside class="contacts">
+    <h2 class="visually-hidden">Контакты</h2>
+    <p class="contacts__intro">Приглашаем
+      к сотрудничеству
+      дилеров!</p>
+    <adress class="contacts__adress">ул. Большая
+      Конюшенная, д. 19/8
+      Санкт-Петербург</adress>
+    <!-- валидатор ругается, говорит в aside нельзя adress вкладывать, хотя из спецификации я это не усмотрела. можно ли так? -->
+    <iframe class="contacts__map"></iframe>
+  </aside>
+  <footer class="page-footer">
+    <a class="page-footer__logo logo">
+      <b class="logo__text">Cat</b> Energy
+    </a>
+    <div class="page-footer__social social">
+      <ul class="social__list">
+        <li class="social__item">
+          <a class="social__link social__link--vkontakte" href="https://vk.com/htmlacademy">
+            <svg></svg>
+            <span class="visually-hidden">Мы в Вконтакте</span>
+          </a>
+        </li>
+        <li class="social__item">
+          <a class="social__link social__link--youtube" href="https://www.youtube.com/user/htmlacademyru">
+            <svg></svg>
+            <span class="visually-hidden">Мы в Ютубе</span>
+          </a>
+        </li>
+        <li class="social__item">
+          <a class="social__link social__link--telegram" href="https://t.me/htmlacademy">
+            <svg></svg>
+            <span class="visually-hidden">Мы в Телеграме</span>
+          </a>
+        </li>
+      </ul>
+    </div>
+    <div class="main-footer__copyright copyright">
+      <a class="copyright__link" href="https://htmlacademy.ru/intensive/adaptive">HTML Academy
+        <svg></svg>
+      </a>
```

Comment:
Тоже, небольшой совет по коду.
Я вот сейчас почти упустил, что внутри ссылки есть и текст, и иконка, потому что когда открывающий и закрывающий тег на отдельных строках, как-то не смотришь есть ли после открывающего тега что-то еще, а смотришь только между тегами.

Поэтому, если уж тег разбит по строкам, то и внутри я бы рекомендовал писать так:
```suggestion
      <a class="copyright__link" href="https://htmlacademy.ru/intensive/adaptive">
        HTML Academy
        <svg></svg>
      </a>
```

Это конечно не правило никакое и как писать в твоем проекте решать тебе. Но мне кажется, что так код поудобнее читается, поэтому просто советую присмотреться =)